.gitignore

```diff
@@ -12,5 +12,6 @@
 *.swo
 *.swm
 
-# VS Code
-/.vscode/
+# VS
+/.vs
```

Comment:
Let's just ignore a `.vs` directory anywhere in the tree, so can we change to `.vs/` here and remove the one we have in `cs\.gitignore`.

Similarly for the `.vscode` directory: `.vscode/`